sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/ColumnarBatch.java

```diff
@@ -232,6 +233,56 @@ public MapData getMap(int ordinal) {
     public Object get(int ordinal, DataType dataType) {
       throw new NotImplementedException();
     }
+
+    @Override
+    public void setNullAt(int ordinal) {
+      columns[ordinal].putNull(rowId);
```

Comment:
Is it better to add checks whether ColumnVector.isConstant == false?
cc @nongli